### ISSUE TITLE
A11y_VSUserStory935073-ContainerToolsWindow_ImagesTab_AI4D: Name Property does not exist for the "Expand/collapse" button.

### ISSUE THREAD

**Issue by gomathip02** [dotnet/wpf#2412]
Copied from VSO ID: 1029463

Bug Description:
“Check out Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.” 

Environment Details:​
Application name-VS User Story 935073-Container Tools Window
Microsoft Visual Studio Enterprise 2019 Int Preview
Version 16.5.0 Preview 1.0 [29505.209.master]
OS: Win10

Repro Steps:​
1.Install and launch the Visual Studio2019 INT preview.
2.Create a project  after checking the "Enable Docker Support" checkbox under "Advanced label".​
3.Debug the solution using F5.​
4. Navigate to View tab->other windows->Containers->Start narrator.​
5.Use the tab key and try navigating to the toolbar below the "Images" tab.
6.Observe if the  Name Property does not exist for the "Expand/collapse" button.

Actual:

Name Property does not exist for the "Expand/collapse" button.

Expected:

Name Property should be there for the "Expand/collapse" button.

Same issue observed for the below scenario:
1.Files tab:
1.Install and launch the Visual Studio2019 INT preview.
2.Create a project  after checking the "Enable Docker Support" checkbox under "Advanced label".
3.Debug the solution using F5.
4. Navigate to View tab->other windows->Containers.
5.Navigate to the Files tab.

User Impact:
Screen Reader users will not understand the context for the "Expand/collapse" button. They would get confused with the names.

Recommendations:​​
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx
​
MAS Reference:
MAS 1.3.1-
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}
MAS4.1.2-
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730}
MAS4.2.1-
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a12f3779-bdf1-40c2-bbcc-0e314ea94858}

**Comment by anjalisheel-wpf** [comment]
@gomathip02 I investigated this bug and I analyzed that this bug is specific to visual studio team and not wpf team as we do not own the code for the "containers" window. So please assign it to the correct team.

**Comment by anjalisheel-wpf** [comment]
@gomathip02 
Please raise this issue in the visual studio forum, find link below :
https://developercommunity.visualstudio.com/search?space=8
